experimenter/experimenter/nimbus-ui/src/components/PageSummary/index.tsx

```diff
@@ -373,3 +381,27 @@ const StatusPill = ({
     {label}
   </Badge>
 );
+
+const AlertWarning = ({
```

Comment:
These warnings are all about launching the experiment, so can we call this `LaunchWarning` ? or maybe just `Warning` ?

Reply:
`Warning` sounds better to me 👍